backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Tests/Controllers/ResourceControllerTest.cs

```diff
@@ -64,6 +65,127 @@ public async Task GetMaskinportenResources_valid()
             AssertionUtil.AssertCollections(expectedResources, actualResources, AssertionUtil.AssertResourceExternalEqual);
         }
 
+        /// <summary>
+        /// Test case: PaginatedSearch with pagination (no search string or filters)
+        /// Expected: PaginatedSearch returns a list of all single rights resources in paginated form with language filtered for the authenticated users selected language
+        /// </summary>
+        [Fact]
+        public async Task GetSingleRightsSearch_searchStringAndFilterNotSet_ReturnsAll()
+        {
+            // Arrange
+            var token = PrincipalUtil.GetToken(1337, 501337, 2);
+            _client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", token);
+            int page = 1;
+            int numPerPage = 7;
+
+            List<ServiceResourceFE> allExpectedResources = TestDataUtil.GetSingleRightsResources();
+            PaginatedList<ServiceResourceFE> expectedResult = new PaginatedList<ServiceResourceFE>(allExpectedResources.GetRange(0, numPerPage), page, allExpectedResources.Count);
+
+            // Act
+            HttpResponseMessage response = await _client.GetAsync($"accessmanagement/api/v1/resources/paginatedSearch?NumPerPage={numPerPage}&Page={page}");
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+            PaginatedList<ServiceResourceFE> actualResources = JsonSerializer.Deserialize<PaginatedList<ServiceResourceFE>>(await response.Content.ReadAsStringAsync(), options);
```

Comment:
I think this line should be moved to under // Act or // Arrange
```suggestion
```

Reply:
Honestly, I just followed the already established convention in the file, but you do have a point that this isn't strictly an Assert.

Though, I'm not sure it fits completely anywhere 😅
It is a reading of the results resulting from an Act so it can't be in Arrange because it happens before Act, but it isn't an Act in itself.

Also, deserializing the results is only possible if the response has a statuscode of OK, thus it makes sense to check for that first so we get an assertion fail rather than a thrown exception and that _is_ in fact an assertion, and... you know what, I've actually argued myself back to thinking this line should remain where it is 😅

Reply:
Yeah it makes sense sequentially and can actually value as we will be able to spot earlier if there are errors related to doing the actual the call. But just looking how it already is in current file or tests isn't necessarily a convention and normally the desirialzing happens in the Act, although it may make sense to have it under Assert as well